html2docx.py

```diff
@@ -6,10 +6,39 @@
 from docx import Document
 from docx.text.paragraph import Paragraph
 from docx.text.run import Run
+from tinycss2 import parse_one_declaration
+from tinycss2.ast import IdentToken
 
 WHITESPACE_RE = re.compile(r"\s+")
 
 
+def attr_to_css(attr: Tuple[str, Optional[str]]) -> Optional[Tuple[str, str]]:
+    if attr[0] == "style":
+        declaration = parse_one_declaration(attr[1])
+        for value in declaration.value:
+            if isinstance(value, IdentToken):
+                return declaration.lower_name, value.lower_value
```

Comment:
This early return assumes there is only a single style. Please include a test with multiple styles. For example:

```html
<span style="font-size:9pt;text-decoration:underline;">Lorem ipsum</span>
```

Reply:
No support for other inline element styles was designed. Only `text-decoration`. Even a case with e. g. subscript and underline will not be handled (support for this is easy to add, if you think it is required).
I added test for two styles combined.